### DISCUSSION TITLE
如何开启稍后读内置解析方案

### DISCUSSION THREAD

**Discussion by seyeeL** [Kenshin/simpread#2773] [使用技巧]
https://es6.ruanyifeng.com/#docs/promise
这个站点在阅读模式下可以解析，但是存到稍后读后解析失败
还有就是这个提示里面的内置解析方案指的是什么，我在帮助文档里也没有搜到关键词

**Comment by Kenshin** [comment]
嗨，你可以这样做：

1. 安装并使用同步助手 https://github.com/Kenshin/simpread/discussions/2754
2. 打开稍后读 → 左下角 

![image](https://user-images.githubusercontent.com/81074/131943962-eb59a795-9aa2-4f55-99c2-cb3b9a30b746.png)

![image](https://user-images.githubusercontent.com/81074/131944000-db6f10c1-dbe7-4879-a4e5-9c17e223fdaa.png)

### 引申

内置解析也不是 100% 可以解析任意内容的，如：需要登录的页面 / 个别页面会组织这种加载方案 等，虽然是少数，占比不会好过 5%，但如果遇到了话，可以使用下面的方式解决。

> [如何让稍后读支持读取本地缓存](https://github.com/Kenshin/simpread/discussions/2146)
